### DISCUSSION TITLE
Ultima: Problem in <p:selectBooleanCheckbox /> CSS when used with itemLabel

### DISCUSSION THREAD

**Discussion by roabol** [primefaces/community#1669] [PrimeFaces Templates] [answered]
I found a problem using the <p:selectBooleanCheckbox component, when used with itemLabel. Notice in the image that the component's border follows the checkbox's label. How can I remove the border that accompanies the itemLabel and leave it just in the checkbox square?

` <p:selectBooleanCheckbox itemLabel="Abrir nova guia" value="#{paginaView.entity.abrirNovaGuia}" />`

![selectBooleanCheckbox](https://github.com/primefaces/community/assets/2237805/a91d111a-ce53-4871-93fb-df48920e13e2)

This happens when I click on the component. I noticed this in theme ultima and barcelona.

**Comment by melloware** [comment]
What version of PF and what version of your themes?
 
cc @jepsar i know he uses the Ultima theme as well.

**Reply by jepsar** [reply]
We use Diamond 🙂

**Reply by melloware** [reply]
@roabol I am not seeing the issue here: https://www.primefaces.org/ultima/uikit/input.xhtml on the Ultima showcase?

**Comment by roabol** [comment]
@melloware,  I'm using version 13.0.7 of PF and version 7.0.0 of ultima.

You don't see the problem in the ultima showcase, because there is no p:selectBooleanCheckbox component, just p:selectManyCheckbox

![image](https://github.com/primefaces/community/assets/2237805/22311ccd-ce0d-478d-b533-5d71ee90be7a)

**Reply by melloware** [reply]
@roabol are you on the Discord channel? If so send me a DM i think I can help.

**Reply by roabol** [reply]
@melloware 

I created a user https://discord.com/. My username is roabol_br, but I'm not familiar with it.

**Reply by melloware** [reply, marked as answer]
For anyone else coming here in the future the trick is to edit this file `\src\main\webapp\resources\layout\styles\theme\extensions\_checkbox.scss`

And comment out the following SCSS...

```css
    &:has(:not(.ui-state-disabled).ui-state-hover) {
      // box-shadow: 0 0 1px 10px rgba($overlayColor, .04);
    }

    &:has(:not(.ui-state-disabled).ui-state-focus) {
      // box-shadow: 0 0 1px 10px rgba($overlayColor, .12);
    }

    &:has(.ui-state-active) {
      &:has(:not(.ui-state-disabled).ui-state-hover) {
        // box-shadow: 0 0 1px 10px rgba($checkboxActiveBg, .04);
      }
  
      &:has(:not(.ui-state-disabled).ui-state-focus) {
        // box-shadow: 0 0 1px 10px rgba($checkboxActiveBg, .12);
      }
    }
```
